code/game/atoms.dm

```diff
@@ -77,6 +77,9 @@
 /atom/proc/emp_act(var/severity)
 	return
 
+/atom/proc/emp_act_console(var/severity)
```

Comment:
Looking at it's [definition](https://github.com/Aurorastation/Aurora/blob/bc126942895bb3c956821b148042c38b20292a17/code/game/mecha/mecha_control_console.dm#L103), it seems like ```in_mecha()``` only ever returns something of type ```/obj/mecha``` or 0. So there doesn't seem to be a reason for this to be an atom proc.